### PR TITLE
[PHP 8.1] Deprecated functions and constants

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

**Comment by kamil-tekiela** [comment]
LGTM

File: reference/image/functions/imagefilledpolygon.xml [hunk 1]
```diff
@@ -82,6 +82,12 @@
      </row>
     </thead>
     <tbody>
+     <row>
+      <entry>8.1.0</entry>
+      <entry>
+       The parameter <parameter>num_points</parameter> has been deprecated.
```
Comment (cmb69) [inline reference/image/functions/imagefilledpolygon.xml:88]: I think that justifies to change the order of the signatures right away, and to mention that the old signature is deprecated.

And the same change needs to be done for imageopenpolygon() and imagepolygon().

File: reference/imap/constants.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
    </term>
    <listitem>
     <simpara>
-     
+     Constant has been deprecated as of PHP 8.1.0.
```
Comment (cmb69) [inline reference/imap/constants.xml:14]: Maybe just:
```suggestion
     Deprecated as of PHP 8.1.0.
```

File: reference/mhash/functions/mhash-count.xml [hunk 3]
```diff
@@ -30,6 +34,30 @@
...
+      <row>
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>8.1.0</entry>
+       <entry>
+        This function has been deprecated. Use the <literal>hash_*()</literal> functions instead.
```
Comment (cmb69) [inline reference/mhash/functions/mhash-count.xml:52]: Maybe link to the [hash function reference](https://www.php.net/manual/en/ref.hash.php)?

File: reference/pdo/constants.xml [hunk 4]
```diff
@@ -333,6 +333,7 @@
     <simpara>
      As <constant>PDO::FETCH_INTO</constant> but object is provided as a serialized string.
      The class constructor is never called if this flag is set.
+     Constant has been deprecated as of PHP 8.1.0.
```
Comment (cmb69) [inline reference/pdo/constants.xml:336]:
```suggestion
     Deprecated as of PHP 8.1.0.
```

**Review by cmb69 — COMMENTED** [review]
Thank you!

**Comment by saundefined** [comment]
Thanks! Updated, but not sure if I did the signature changes right 🤔

**Comment by cmb69** [comment]
Thank you! I've applied this with a few tweaks.

**Pull request closed without merging**